crates/typst-library/src/visualize/image.rs

```diff
@@ -64,7 +68,77 @@ pub struct ImageElem {
     pub fit: ImageFit,
 }
 
-impl Layout for ImageElem {
+/// A raster or vector graphic loaded from string.
+///
+/// Supported formats are PNG, JPEG, GIF and SVG.
+///
+/// _Note:_ Work on SVG export is ongoing and there might be visual inaccuracies
+/// in the resulting PDF. Make sure to double-check embedded SVG images. If you
+/// have an issue, also feel free to report it on [GitHub][gh-svg].
+///
+/// ## Example { #example }
+/// ```example
+/// #figure(
+///   image.decode(```<svg xmlns="http://www.w3.org/2000/svg" height="140" width="500"><ellipse cx="200" cy="80" rx="100" ry="50" style="fill:yellow;stroke:purple;stroke-width:2" /></svg>```.text, "svg", width: 80%),
```

Comment:
I think the example with the very long line will not look that good in the docs. Maybe we could show another use case like loading an SVG from a file, but making some string replacements?

Reply:
I put this example into the tests and replaced it by a loaded file.